modules/alloy/src/main/ts/ephox/alloy/ui/composite/TypeaheadSpec.ts

```diff
@@ -277,7 +277,11 @@ const make: CompositeSketchFactory<TypeaheadDetail, TypeaheadSpec> = (detail, co
         sandbox: (hotspot) => {
           return DropdownUtils.makeSandbox(detail, hotspot, {
             onOpen: () => Toggling.on(hotspot),
-            onClose: () => Toggling.off(hotspot)
+            onClose: () => {
+              // TINY-9280: Remove aria-activedescendant that is set when menu item is highlighted
+              detail.lazyTypeaheadComp.get().each((input) => Attribute.remove(input.element, 'aria-activedescendant'));
+              Toggling.off(hotspot);
+            }
```

Comment:
`TieredMenuDetail` has a parameter `onDehighlightItem` which could be used rather than this. Since you are using `onHighlightItem` to set the attribute.

Reply:
`onDehilightItem` is only called when another item is highlighted. The aria attribute wont be removed when the menu is closed

Reply:
Agreed to include comments clarifying why the placement of these functions to apply/remove attributes are in separate files

Reply:
Comment is updated